IntentToSecondScreen/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright (c) Microsoft Corporation. All rights reserved.
+  ~ Licensed under the MIT License.
+  ~
+  -->
+
+<resources>
+    <string name="intent_app_name">Absicht zum zweiten Bildschirm</string>
```

Comment:
It can work, but like others, make sure that we go for an unified approach for samples-names (English or German).